### PR TITLE
Add github default labels to the label configuration

### PR DESCRIPTION
**Pull request by cblecker**
This would cause labels for new repos to get migrated into the ones that we use in the org.

/hold
Explicit hold as this requires communication and consensus per ContribEx policy.
cc: @kubernetes/sig-contributor-experience-pr-reviews

### CODE REVIEW

**Comment by cblecker** [comment]
This will have to wait until after https://github.com/kubernetes/test-infra/pull/8128, as I will need to rebase.

File: label_sync/labels.yaml [hunk 1]
```diff
@@ -49,11 +50,22 @@ labels:
       - name: close/not-reproducible
     target: both
     addedBy: humans
+  - color: d455d0
+    description: Indicates an issue that is an support question that was opened incorrectly. The usage should be supported by adding triage findings as an issue comment.
```
Comment (spzala) [inline label_sync/labels.yaml]: Nit - replace "an support question" to "a support question"? OR change the sentence to "Indicates an issue is a support question that was opened incorrectly."?

File: label_sync/labels.md [hunk 2]
```diff
@@ -25,15 +25,16 @@ larger set of contributors to apply/remove them.
...
 | <a id="triage/needs-information" href="#triage/needs-information">`triage/needs-information`</a> | Indicates an issue needs more information in order to work on it. The usage should be supported by adding triage findings as an issue comment. <br><br> This was previously `close/needs-information`, | humans | |
 | <a id="triage/not-reproducible" href="#triage/not-reproducible">`triage/not-reproducible`</a> | Indicates an issue can not be reproduced as described. The usage should be supported by adding triage findings as an issue comment. <br><br> This was previously `close/not-reproducible`, | humans | |
-| <a id="triage/unresolved" href="#triage/unresolved">`triage/unresolved`</a> | Indicates an issue that can not be resolved. The usage should be supported by adding triage findings as an issue comment. <br><br> This was previously `close/unresolved`, | humans | |
+| <a id="triage/support" href="#triage/support">`triage/support`</a> | Indicates an issue that is an support question that was opened incorrectly. The usage should be supported by adding triage findings as an issue comment. <br><br> This was previously `close/support`, `kind/support`, `question`, | humans | |
```
Comment (spzala) [inline label_sync/labels.md]: Nit - replace "an support question" to "a support question" ? OR change the sentence to "Indicates an issue is a support question that was opened incorrectly."? that will be more aligned with other sentences (e.g. LOC 28 above)

File: label_sync/labels.yaml [hunk 3]
```diff
@@ -43,11 +44,22 @@ labels:
       - name: close/not-reproducible
     target: both
     addedBy: humans
+  - color: d455d0
+    description: Indicates an issue that is a support question that was opened incorrectly.
```
Comment (cblecker) [inline label_sync/labels.yaml]: I'm going to remove the incorrectly part. This is true for k/k, but not across the org.

**Comment by cblecker** [comment]
I'm also adding label entries for kind/cleanup, kind/flake, and kind/design. We already use these, they just weren't defined. (See https://github.com/kubernetes/community/issues/2032)

**Comment by cblecker** [comment]
/hold cancel
This has been communicated out, and can be merged at any time.

/assign @cjwagner @BenTheElder 
Can I get an LGTM? :)

**Comment by BenTheElder** [comment]
punting to oncall @cjwagner to merge / deploy. changes seem fine to me :-)

**Comment by fejta** [comment]
/lgtm

No need to involve oncall in this sort of thing. Config is deployed automatically.

**Comment by BenTheElder** [comment]
whoops, there's a related PR as well adding kubernetes-sigs that needs to be deployed. I got interrupted and came back to respond to both.

**Comment by BenTheElder** [comment]
https://github.com/kubernetes/test-infra/pull/8218

**Pull request merged**